src/sidebar/app/panel.js

```diff
@@ -0,0 +1,288 @@
+/* exported TEXT_ALIGN_DIR */
```

Comment:
Should we just delete this file? It looks 100% commented out.

Reply:
I did split content between Editor and Footer component, commenting section by section while progressing. Main reason I kept it is because the TEXT_ALIGN_DIR seams not used anymore and I want to do more test. Will be removed yes.